recipe/meta.yaml

```diff
@@ -1,17 +1,26 @@
 {% set name = "numcodecs" %}
-{% set version = "0.13.1" %}
+{% set version = "0.14.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a3cf37881df0898f3a9c0d4477df88133fe85185bffe57ba31bcc2fa207709bc
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f138f5e4f574a94a4a8ba1c8be9323ec89bce363f4c65801d8e29e5064c6599
+  patches:
+    ################################################################
+    # Fix softening of `crc32c` test dependency                    #
+    #                                                              #
+    # xref: https://github.com/zarr-developers/numcodecs/pull/635  #
+    # xref: https://github.com/zarr-developers/numcodecs/pull/637  #
+    ################################################################
+    - PR_635.patch
+    - PR_637.patch 
```

Comment:
Applying upstream PRs as patches to see if this fixes the CI error

xref: https://github.com/zarr-developers/numcodecs/pull/635
xref: https://github.com/zarr-developers/numcodecs/pull/637